index.js

```diff
@@ -64,6 +64,7 @@ app.use('/typecheck', require('./vulnerabilities/typecheck'));
 app.use('/mongoose', require('./vulnerabilities/mongoose'));
 app.use('/express-session', require('./vulnerabilities/express-session'));
 app.use('/ddb', require('./vulnerabilities/dynamodb'));
```

Comment:
We built the test bench utils and such to normalize a lot across frameworks. I get this is a 1 off but this sink still applies to xss. Can you follow the `/<rule>/<sink>/unsafe|safe` model? Feels like this work is also out of scope and should be its own ticket to do right and not just rushed to show this sink works functionally

Reply:
If that's team consensus, I wouldn't mind declining this PR and deferring to a ticket.

Reply:
works for me.